### ISSUE TITLE
S3 bucket notification configuration with multiple LambdaFunctionConfigurations

### ISSUE THREAD

**Issue by JasonLarson44** [localstack/localstack#1265]
Hi guys,

This issue is related to #450, which has been closed, but I am still seeing this issue. 
I am spinning up localstack using docker-compose with localstack:0.9.0

I can successfully put a bucket notification if I remove one of the configurations. 

When I try to add a configuration with multiple LambdaFunctionConfigurations I get this error:
```
2019-04-22T21:13:17:ERROR:localstack.services.generic_proxy: Error forwarding request: 'list' object has no attribute 'get' Traceback (most recent call last):
File "/opt/code/localstack/localstack/services/generic_proxy.py", line 196, in forward
    path=path, data=data, headers=forward_headers)
File "/opt/code/localstack/localstack/services/s3/s3_listener.py", line 459, in forward_request
    response = handle_notification_request(bucket, method, data)
File "/opt/code/localstack/localstack/services/s3/s3_listener.py", line 391, in handle_notification_request
    events = config.get('Event')
AttributeError: 'list' object has no attribute 'get'
```
My Config File:
```
{
    "LambdaFunctionConfigurations": [
        {
            "Id": "50c2390e-4698-4563-8cd8-d7545532c284",
            "LambdaFunctionArn": "arn:aws:lambda:us-west-2:000000000000:function:function1",
            "Events": [
                "s3:ObjectCreated:*"
            ],
            "Filter": {
                "Key": {
                    "FilterRules": [
                        {
                            "Name": "suffix",
                            "Value": ".yml"
                        },
                        {
                            "Name": "prefix",
                            "Value": "prefix1/"
                        }
                    ]
                }
            }
        },
        {
            "Id": "50c2390e-4698-4563-8cd8-d7545532c285",
            "LambdaFunctionArn": "arn:aws:lambda:us-west-2:100000000000:function:function2",
            "Events": [
                "s3:ObjectCreated:*"
            ],
            "Filter": {
                "Key": {
                    "FilterRules": [
                        {
                            "Name": "suffix",
                            "Value": ".json"
                        },
                        {
                            "Name": "prefix",
                            "Value": "prefix2/"
                        }
                    ]
                }
            }
        }
    ]
}
```

Thanks!

**Comment by fbn-roussel** [comment]
Hello,

Any news on this issue ? 

Thank you.

**Comment by mascah** [comment]
Same issue exists when using multiple `TopicConfigurations` as well.

**Comment by whummer** [comment]
Thanks @JasonLarson44 @fbn-roussel @mascah . Similiar issue has been reported in #450. This should be fixed in #1716 - can you please give it a try with the latest version of the Docker image? Please re-open this issue if the problem persists. Thanks!
